### ISSUE TITLE
Tradfri spots losing connection shortly after pairing

### ISSUE THREAD

**Issue by repvik** [dresden-elektronik/deconz-rest-plugin#937]
I recently went from 3 to 12 trådfri GU10 spots on my raspbee. The original three spots and three of the new spots can now be paired easily, some of them may work a couple of minutes after being paired. After a few minutes, all the spots are grayed out in the webinterface, and I can no longer control them from deconz. 
Resetting the bulbs and pairing again exhibits the same issue. Once communication with a spot has been lost, I can _maybe_ restore it (for a few minutes) by re-pairing it. Power-cycling RPI and bulbs make no difference. 
The remaining 6 new spots work flawlessly, as well as 4 regular bulbs further away from the RPI. 

Running 2.05.44/262D0500, Phoscon says this is up to date.

**Comment by repvik** [comment]
I've got problems with one of these:
TRADFRI bulb GU10 W 400lm - Version 1.2.214
Two identical spots, same firmware, are working. They are right next to the problematic spot, and were bought the same day.

and I've got problems with three of these:
TRÅDFRI bulb GU10 WS 400lm - Version 1.2.217
Only one of these three spots show a fw version. None of them work, however. These worked fine before I rearranged the lights and re-paired them. 

As of right now, I'm running home-assistant, and do not have access to the GUI. I'll have to make a sd card to check the NUI.
